.travis.yml

```diff
@@ -56,6 +56,8 @@ matrix:
     - python: 2.7
       env:
         - INSTALL_TYPE=wheel
+        # Dependency checking should get all needed dependencies
+        - DEPENDS=
```

Comment:
You're right - that's annoying.   I'm maxed out on travis-ci at the moment - would you mind trying a branch where the lines are `DEPENDS=""` to see if that takes?